api/py/ai/chronon/repo/join_backfill.py

```diff
@@ -106,15 +106,17 @@ def run_join_part(self, join_name: dict, join_part: str):
         )
 
     def run_left_table(self, join_name: str):
-        settings = self.settings.get("left_table", self.settings["default"])
+        # Merge default settings and left_table settings
+        settings = {**self.settings["default"], **self.settings.get("left_table", {})}
         return (
             self.export_template(settings)
             + " && "
             + self.command_template(config_path=get_config_path(join_name), extra_args={"mode": "backfill-left"})
         )
```

Comment:
do we need to do this for join part backfill?

Reply:
Good catch!